recipes/imagl/all/conanfile.py

```diff
@@ -95,31 +96,27 @@ def lazy_lt_semver(v1, v2):
             print("Your compiler is {} {} and is compatible.".format(str(self.settings.compiler), compiler_version))
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  destination=self._source_subfolder, strip_root=True)
-
-    def _configure_cmake(self):
-        if self._cmake:
-            return self._cmake
-
-        self._cmake = CMake(self)
-
-        self._cmake.definitions["STATIC_LIB"] = not self.options.shared
-        self._cmake.definitions["SUPPORT_PNG"] = self.options.with_png
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
```

Comment:
This can be simplified.

```suggestion
        get(self, **self.conan_data["sources"][self.version], strip_root=True)
```